### PR TITLE
Work in progress

### PR DESCRIPTION
**Pull request by atribolet**
Pull request description
------------------------

### CODE REVIEW

**Comment by steveoh** [comment]
> So I'm still not finished editing the article, but here are a few more surface edits. I think now that I'm starting to get more familiar with editing in GitHub, I'll focus on enhancing the style, clarifying the wording, etc. with my edits.

Sounds great!

When you are happy with your work, rename the pull request 

image

and leave a comment so we get notified.

thanks

**Comment by atribolet** [comment]
Oh my gosh, Leslie Knope is my hero! lol

File: _posts/2018-02-26-success-with-serverless.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ tags:
...
+This quad-word system works great . . . until you try to use one of Esri's out-of-the-box print services ([here's an example of one](http://mapserv.utah.gov/arcgis/rest/services/Utilities/PrintingTools/GPServer)). When you send a web map to one of these print services, the service reconstructs all of the layers on the server. This causes requests for base maps to be sent from your ArcGIS Server box rather than your user's browser (with your domain as the referrer). Now, we do allow wide-open quad-words (i.e., quad-words not locked down to any domain/IP) to be used by those who need to make requests from servers or other local machines. However, the wide-open quad-words can't be used in web applications because they could be copied and used by unauthorized users. This is a problem.
```
Comment (atribolet) [inline _posts/2018-02-26-success-with-serverless.md]: _Now, we do allow wide-open quad-words (i.e., quad-words not locked down to any domain/IP) to be used by those who need to make requests from servers or other local machines_.: Did I understand the "wide-open quad-words" thing correctly? Is this rewording accurate?

File: _posts/2018-02-26-success-with-serverless.md [hunk 2]
```diff
@@ -28,15 +28,17 @@ While we did make this solution work, it was not ideal. Geoprocessing services,
 In the end, we decided that there was no need to incur all of the heavy overhead of ArcGIS Server for a simple proxy service.
 
 ### Serverless to the Rescue
+This is where serverless computing came in.
```
Comment (atribolet) [inline _posts/2018-02-26-success-with-serverless.md]: I feel like we needed another line here. What do we think?

**Comment by steveoh** [comment]
here's the [website preview](https://deploy-preview-793--gis-utah-gov.netlify.com/success-with-serverless/) link for ease of finding!

File: _posts/2018-02-26-success-with-serverless.md [hunk 3]
```diff
@@ -28,15 +28,17 @@ While we did make this solution work, it was not ideal. Geoprocessing services,
...
 
-Many of the major vendors provide command-line utilities to help you get up and running quickly. We decided to give the provider-agnostic project, [serverless](https://serverless.com) a try. [Getting started](https://serverless.com/framework/docs/providers/google/guide/quick-start/) was fairly simple:
+Many of the major vendors provide command-line utilities to help you get up and running quickly. We decided to give the provider-agnostic project, [Serverless](https://serverless.com), a try. [Getting started](https://serverless.com/framework/docs/providers/google/guide/quick-start/) was fairly simple:
```
Comment (atribolet) [inline _posts/2018-02-26-success-with-serverless.md]: _We decided to give the provider-agnostic project, [Serverless](https://serverless.com), a try.:_ "Serverless" in this sentence is referring to the proper name of a product, right? If so, I think we should cap it.

File: _posts/2018-02-26-success-with-serverless.md (hunk 1 above)
Comment (steveoh) [reply _posts/2018-02-26-success-with-serverless.md]: yes that is accurate.

File: _posts/2018-02-26-success-with-serverless.md (hunk 3 above)
Comment (steveoh) [reply _posts/2018-02-26-success-with-serverless.md]: correct. their logo is lowercase but they use proper case through out their website.

Comment (atribolet) [reply _posts/2018-02-26-success-with-serverless.md]: Ok that's what I thought! Good good.

**Comment by atribolet** [comment]
Ok, I think I'm finished editing unless we need any other wording changes? Just let me know!

**Comment by stdavis** [comment]
LGTM

**Pull request merged**